hw/mips/malta.c

```diff
@@ -1220,6 +1228,24 @@ static void mips_create_cpu(MachineState *ms, MaltaState *s,
     }
 }
 
+static void create_virtio_devices(void)
+{
+    CPUMIPSState *env;
+    MIPSCPU *cpu;
+
+    cpu = MIPS_CPU(first_cpu);
+    env = &cpu->env;
+    for(int i = VIRTIO_N_TRANSPORTS-1; i >= 0; i--) {
```

Comment:
```suggestion
    for (int i = VIRTIO_N_TRANSPORTS-1; i >= 0; i--) {
```

Reply:
Also, why is this counting down?!

Reply:
This makes the order of addresses of virtio devices match the command line order (something reverses it somewhere). Obviously, systems should be probing for devices rather than making any assumptions, but I just think its neater that if you have one device, it appears at the lowest address etc.